src/api/helpers/map-address/index.ts

```diff
@@ -0,0 +1,19 @@
+import { OrdnanceSurveyResponse } from '../../types';
+
+/**
+ * mapAddress
+ * Maps address from OrdnanceSurveyResponse to correct format
+ * @param {OrdnanceSurveyResponse} address
+ * @returns {Address} mapped address
+ */
+const mapAddress = (address: OrdnanceSurveyResponse) => ({
+  addressLine1: `${address.DPA.ORGANISATION_NAME ?? ''} ${address.DPA.BUILDING_NAME ?? ''} ${address.DPA.BUILDING_NUMBER ?? ''} ${
+    address.DPA.THOROUGHFARE_NAME ?? ''
+  }`.trim(),
+  addressLine2: address.DPA.DEPENDENT_LOCALITY,
+  town: address.DPA.POST_TOWN,
+  county: undefined,
```

Comment:
I don't understand why this is needed, the type is `county?: string;`, if there is a lint error without this, maybe need to add a return type to the function
```js
const mapAddress = (address: OrdnanceSurveyResponse): Address => ({
```

Reply:
Just tested this, will push. All works fine without adding undefined values:

```js
addressLine2: undefined,
county: undefined,
county: undefined,
```